classes/settings_provider.php

```diff
@@ -58,6 +58,11 @@ class settings_provider {
      */
     const USE_SEB_CLIENT_CONFIG = 4;
 
+     /**
+      * The filearea itemid which is used for saving the module template.
```

Comment:
Is this doc correct?  are we using it for template?

Reply:
Nice one, I'll adjust that.